app/utils/IncidentP5Helper.scala

```diff
@@ -163,6 +163,32 @@ class IncidentP5Helper(
     ).flatten
   )
 
+  def containerIdentificationNumberRow(equipmentIndex: Index): Option[SummaryListRow] =
```

Comment:
Can the transport equipment stuff live in its own helper? So it will have a `TransportEquipmentType07`